wave_containers/include/wave/containers/measurement_container.hpp

```diff
@@ -61,8 +61,8 @@ struct measurement_container {
     struct sensor_key : member<T, decltype(T::sensor_id), &T::sensor_id> {};
 
     // Define a composite key which combines the above two keys
-    // This lets us search elements sorted by sensor id first, then time
-    struct sensor_and_time_key : composite_key<T, sensor_key, time_key> {};
+    // This lets us search elements sorted by time first, then  sensor id
+    struct sensor_and_time_key : composite_key<T, time_key, sensor_key> {};
```

Comment:
How likely will it be for two measurements to have the same time_point but different sensor_id?

Seems pretty likely especially if the user does something like this
```cpp
time_now = getSystemTime();
m.emplace(time_now, sensor1, pollSensor1());
m.emplace(time_now, sensor2, pollSensor2());
```